src/utils/crypt.py

```diff
@@ -29,5 +30,8 @@
 
 from hashlib import md5
 from hashlib import sha1 as sha
+from hashlib import sha256 as sha_256
+from hashlib import sha512 as sha_512
+from hashlib import ripemd160 as ripemd_160
```

Comment:
`ripemd160` is not guaranteed to be available on all Python systems, see [hashlib's documentated](https://docs.python.org/2/library/hashlib.html)